tests/test_awslambda/test_lambda.py

```diff
@@ -236,6 +258,69 @@ def test_create_function_from_image():
     code.should.have.key("ResolvedImageUri").equals(resolved_image_uri)
 
 
+@mock_lambda
+def test_create_function_from_mocked_ecr(
```

Comment:
Env vars cannot (easily) be set in ServerMode, so let's just skip the two new tests there:
```
if settings.TEST_SERVER_MODE:
    raise SkipTest("..")
```
Some tests in `test_lambda_invoke.py` follow the same pattern, such as `test_invoke_lambda_using_environment_port`. 
(Just note: that test is the other way around, it _only_ runs in ServerMode - we want to _not_ run in ServerMode)